### DISCUSSION TITLE
Geolocation failed

### DISCUSSION THREAD

**Discussion by vxkShelby** [plan-player-analytics/Plan#3509] [Q&A]
can i somehow fix this error in console ? 

09.03 12:07:08 [Server] WARN [Plan] Downloading GeoLite2 requires accepting GeoLite2 EULA - see 'Data_gathering.Accept_GeoLite2_EULA' in the config.
09.03 12:07:08 [Server] ERROR [Plan] Failed to enable geolocation.

**Comment by AuroraLS3** [comment]
Yes it's fixed in this dev release https://github.com/plan-player-analytics/Plan/releases/tag/5.6.2796

If all goes well there's a full release tomorrow, if not, next weekend.

**Comment by vxkShelby** [comment]
tnx

**Reply by vxkShelby** [reply]
miss

**Comment by vxkShelby** [comment]
i can see that i have same dev release but still error ...

**Reply by AuroraLS3** [reply]
You need to read & accept the MaxMind EULA (See the config for Data_gathering.Accept_GeoLite2_EULA setting, there's linke there above the setting)

**Reply by vxkShelby** [reply]
where can i find that

**Reply by vxkShelby** [reply]
i got it .... tnx

**Reply by vxkShelby** [reply]
after accepting i still getting this : 

28.03 10:51:29 [Server] WARN [Plan] Downloading GeoLite2 requires accepting GeoLite2 EULA - see 'Data_gathering.Accept_GeoLite2_EULA' in the config.
28.03 10:51:29 [Server] ERROR [Plan] Failed to enable geolocation.

**Reply by AuroraLS3** [reply]
You need to set the setting to true

**Reply by vxkShelby** [reply]
lol i did it but it was false now ... after plugin reload it seems okay ... tnx
